### ISSUE TITLE
Merak Network: Check network info from scenario manager got error

### ISSUE THREAD

**Issue by cj-chung** [futurewei-cloud/merak#76]
Scenario action on network to check network info, the following error return from network:

```
{"Service":"scenario-manager","level":"info","msg":"constructNetConfMessage: config:{format_version:1  revision_number:1  request_id:\"5f85f79d5b0a4d33a47c63442d91cf0a\"  netconfig_id:\"e6b7e72ac496490e8e4d9127798ec802\"  network:{id:\"e6b7e72ac496490e8e4d9127798ec802\"  name:\"network-config-1\"  number_of_vpcs:1  number_of_subnet_per_vpc:1  vpcs:{tenant_id:\"123456789\"  project_id:\"123456789\"  subnets:{subnet_cidr:\"10.0.1.0/20\"  subnet_gw:\"10.0.1.1\"  number_vms:100}}  number_of_security_groups:1  routers:{name:\"string\"  subnets:\"string\"}  gateways:{name:\"string\"  ips:\"string\"}  security_groups:{name:\"sg-1\"  rules:{name:\"string\"  description:\"string\"  ethertype:\"string\"  protocol:\"string\"  port_range:\"string\"  remote_group_id:\"string\"  remote_ip_prefix:\"string\"}  apply_to:\"string\"}}}","time":"2022-08-22T17:48:01Z"}
{"Service":"scenario-manager","level":"error","msg":"Error when calling Merak-Network: rpc error: code = Unknown desc = redis: nil","time":"2022-08-22T17:48:01Z"}
{"Service":"scenario-manager","level":"error","msg":"error return from grpc server: error when calling merak-network grpc server: rpc error: code = Unknown desc = redis: nil","time":"2022-08-22T17:48:01Z"}
{"Service":"scenario-manager","level":"error","msg":"'CHECK' network failed: deploy network failed, Error = 'error return from grpc server: error when calling merak-network grpc server: rpc error: code = Unknown desc = redis: nil'","time":"2022-08-22T17:48:01Z"}
```

**Comment by yanmo96** [comment]
I think it is a k8s config issue.  
The service will point to the compute db sometimes.  
Just update the k8s yaml on 77. After test will push a PR.

**Comment by yanmo96** [comment]
This is fixed in PR #79 .  
Issue been the redis/selector/app is the same name as the Merak Compute. 
Closing this issue now.
